### ISSUE TITLE
No font choice in theme editor

### ISSUE THREAD

**Issue by fefa** [bhadaway/support#7]
Maybe I'm blind, but I can't see the font family choice menu? 

![Screenshot 2023-06-16 120506](https://github.com/bhadaway/support/assets/6717095/dbd7d4f7-c5e4-4ab1-bf1d-c9df60b79e15)

**Comment by bhadaway** [comment]
Which theme are you using?

**Comment by fefa** [comment]
Sorry, it's Block Builder

**Comment by bhadaway** [comment]
Things to try:

1. Highlight the text first
2. Click on "Aa"
3. Click on "..." next to Typography

**Comment by fefa** [comment]
Yes tried that, no font family there:

![Screenshot 2023-06-16 155425](https://github.com/bhadaway/support/assets/6717095/78408bbe-7eeb-4a4a-b1ed-26a78f87ba8b)

**Comment by fefa** [comment]
Just to make it clear: I can change font when editing a page no problem, what I am trying to do is change a default theme font so that I don't have to manually set the font on every post.

**Comment by bhadaway** [comment]
I see. Block Builder is meant to be a blank slate so there is no default font set, which might be the problem. Let me take a look and see what the best solution is...

**Comment by fefa** [comment]
Right, that's why I chose it :)
So then the question is how you set the font, I was looking for something like:
![Screenshot 2023-06-16 161720](https://github.com/bhadaway/support/assets/6717095/a00435bd-f76f-4280-b7d0-f30041aba240)

Can try to find a plugin that does that.

**Comment by fefa** [comment]
Right, CSS it is then - I was hoping to avoid having to change URLs after I move to proper hosting.
Thanks a lot for quick replies and the update!
